Sources/Error Handling/ErrorCode.swift

```diff
@@ -59,7 +58,19 @@ import Foundation
     @objc(RCInvalidPromotionalOfferError) case invalidPromotionalOfferError = 34
     @objc(RCOfflineConnectionError) case offlineConnectionError = 35
 
+    /**
+     * When an ErrorCode has been deprecated and then removed, add it to the reserved list so that we do not
+     * accidentally reuse it. For example:
+     * `@objc(RCMissingAppUserIDForAliasCreationError) case missingAppUserIDForAliasCreationError = 27` was removed,
+     * so we add its rawValue of `27` to the `reservedRawValues` array. That way our unit tests will catch if we
+     * accidentally add `27` back into the enumeration.
+     */
+    static var reservedRawValues: [Int] {
```

Comment:
Nit: maybe move this to an extension?

Reply:
I love this idea though.

Reply:
Should be a set though, and maybe use `RawValue` instead?
```suggestion
    static var reservedRawValues: Set<RawValue> {
```

Reply:
✅